src/maidsafe/vault/mpid_manager/database.cc

```diff
@@ -59,7 +57,183 @@ MpidManagerDataBase::~MpidManagerDataBase() {
   }
 }
 
+void MpidManagerDatabase::Put(const MpidManager::MessageKey& key,
+                              const uint32_t size,
+                              const MpidManager::GroupName& group_name) {
+  if (!db_)
+    BOOST_THROW_EXCEPTION(MakeError(CommonErrors::db_not_present));
+  CheckPoint();
+
+  sqlite::Transaction transaction{*db_};
+  std::string query(
+      "INSERT OR REPLACE INTO MpidManagerAccounts (Chunk_Name, Chunk_size,"
+      " MPID) VALUES (?, ?, ?)");
+  sqlite::Statement statement{*db_, query};
+
+  std::string mpid(EncodeGroupName(group_name));
+  statement.BindText(3, mpid);
+  std::string chunk_size(std::to_string(size));
```

Comment:
We should probably be using our Cereal helper functions in [serialisation.h](https://github.com/Fraser999/MaidSafe-Common/blob/next/include/maidsafe/common/serialisation/serialisation.h) here to be more efficient and maybe more importantly, consistent in our approach to serialisation.  Running this:
```
  std::cout << "Size of std::to_string(0):            " << std::to_string(0).size() << '\n';
  std::cout << "Size of std::to_string(1024 * 1024):  " << std::to_string(1024 * 1024).size() << '\n';
  std::cout << "Size of ConvertToString(0):           " << ConvertToString(0).size() << '\n';
  std::cout << "Size of ConvertToString(1024 * 1024): " << ConvertToString(1024 * 1024).size() << '\n';
```
yields the following output:
```
Size of std::to_string(0):            1
Size of std::to_string(1024 * 1024):  7
Size of ConvertToString(0):           4
Size of ConvertToString(1024 * 1024): 4
```